src/XML/src/FMI3/fmi3_xml_model_description.c

```diff
@@ -69,9 +65,19 @@ fmi3_xml_model_description_t * fmi3_xml_allocate_model_description( jm_callbacks
     md->defaultExperiment.stepSizeDefined  = 0;
 
     md->defaultExperiment.startTime = 0;
-    md->defaultExperiment.stopTime = 1.0;
+    md->defaultExperiment.stopTime  = 1.0;
     md->defaultExperiment.tolerance = FMI3_DEFAULT_EXPERIMENT_TOLERANCE;
-    md->defaultExperiment.stepSize = FMI3_DEFAULT_EXPERIMENT_STEPSIZE;
+    md->defaultExperiment.stepSize  = FMI3_DEFAULT_EXPERIMENT_STEPSIZE;
+
+    jm_vector_init(char)(&md->modelExchange.modelIdentifier, 0, cb);
+
+    jm_vector_init(char)(&md->coSimulation.modelIdentifier, 0, cb);
+    md->coSimulation.hasFixedInternalStepSize = 0;
+    md->coSimulation.fixedInternalStepSize    = 0.0;
+    md->coSimulation.hasRecommendedIntermediateInputSmoothness = 0;
```

Comment:
According to FMI spec there is a default value of 0. Do we still need this variable?